spec/index.bs

```diff
@@ -2886,6 +2886,7 @@ enum GPUFeatureName {
     "clip-distances",
     "dual-source-blending",
     "subgroups",
+    "read-write-storage-texture-format-tier2",
```

Comment:
Maybe we can just land this as-is, but named "storage-texture-format-tier2", and then in a followup we can decide whether to merge tier 3 into this extension or make a new one?

Given `shaderStorageImageExtendedFormats` is on almost all devices I am fine with merging tier 3. I just want a paper trail on the appropriate PR for how we decided they would be readonly.

Reply:
Do you mean the below statement in the [Vulkan SPEC](https://registry.khronos.org/vulkan/specs/latest/html/vkspec.html#descriptorsets-storageimage):

Storage image loads are supported in all shader stages for image views whose format features contain VK_FORMAT_FEATURE_STORAGE_IMAGE_BIT.

Reply:
Sorry, I realized I completely misread the spec text for tier 3, I thought it was saying that the R formats didn't support read-write, but I couldn't understand why. But actually it was saying:
- The RGBA formats already have read-only and write-only, we're adding read-write
- The R formats don't have storage at all, we're adding read-only, read-write, _and_ read-write

Then my only comment is that if we add back tier 3, we make that a bit clearer. Apologies!